bin/test

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+# Hack per:
+# http://stackoverflow.com/questions/4774054/reliable-way-for-a-bash-script-to-get-the-full-path-to-itself
+pushd $(dirname $0) >/dev/null
+PACKAGE_ROOT=$(dirname $(pwd -P))
+popd >/dev/null
+
+export HUBOT_SLACK_GITHUB_ISSUES_CONFIG_PATH="${PACKAGE_ROOT}/test/helpers/test-config.json"
+exec ${PACKAGE_ROOT}/node_modules/hubot/bin/hubot
```

Comment:
Yeah, it's just something to run Hubot to make sure the script loads and doesn't explode when you type text. Will probably name it something different eventually.